tools/tpm2_ptool.py

```diff
@@ -871,7 +883,9 @@ def do_token_init(db, path, args):
         # Verify pid is in db
         pobject = db.getprimary(pid)
 
-        pobjauthhash = AddTokenCommand.verify_pobjpin(pobject, pobjpin)
```

Comment:
an empty pobjpin should be passing "" here, is it?

Reply:
@williamcroberts right again.